js/tests/unit/tab.spec.js

```diff
@@ -548,6 +548,62 @@ describe('Tab', () => {
       expect(Event.prototype.stopPropagation).toHaveBeenCalledTimes(2)
       expect(Event.prototype.preventDefault).toHaveBeenCalledTimes(2)
     })
+
+    it('if keydown event is right arrow and next element has disabled attribute', () => {
+      fixtureEl.innerHTML = [
+        '<div class="nav">',
+        '  <span id="tab1" class="nav-link" data-bs-toggle="tab"></span>',
+        '  <span id="tab2" class="nav-link" data-bs-toggle="tab" disabled></span>',
```

Comment:
you may can add `disabled` class here and do the same checks for **left** arrow on next test.
 _( We just have to test left arrow too, the way, it's up to your preference.)_

Reply:
I've duplicated the right arrow tests for the left arrow in https://github.com/twbs/bootstrap/pull/36169/commits/153ee5f3d89185786bd78a4d444fdc26befdcff8.
I thought it would be interesting to have separate tests for `disabled` attribute and `.disabled` in order to highlight the fact that the interaction can work with one or the other. But it is maybe a mistake. I can gather them into 2 tests if you prefer where I mix the `disabled` attribute and `.disabled`—something like:
```html
<span id="tab2" class="nav-link disabled" data-bs-toggle="tab" disabled></span>'
```

Reply:
Excuse me, I may wan't clear in the previous comment
I was suggesting something like:


```html

 fixtureEl.innerHTML = [
        '<div class="nav">',
        '  <span id="tab1" class="nav-link" data-bs-toggle="tab"></span>',
        '  <span id="tab2" class="nav-link" data-bs-toggle="tab" disabled></span>', // attribute
        '  <span id="tab2" class="nav-link disabled" data-bs-toggle="tab"></span>', /// class
        '  <span id="tab3" class="nav-link" data-bs-toggle="tab"></span>',
        '</div>'
      ].join('')


```
not 

```html

<span id="tab2" class="nav-link disabled" data-bs-toggle="tab" disabled></span>'
```
 
having in mind to caver both cases, in one test

Reply:
Fixed in https://github.com/twbs/bootstrap/pull/36169/commits/543493f7dc98a030996311a9cc44e27e008050f1 + https://github.com/twbs/bootstrap/pull/36169/commits/f26cb853b5df7c2e97c9bdea6ed6a853c344d401